paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -385,51 +392,22 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
         }
     }
 
-    fun handlePrimaryButtonClick(screenState: USBankAccountFormScreenState) {
-        when (screenState) {
-            is USBankAccountFormScreenState.BillingDetailsCollection -> {
-                _currentScreenState.update {
-                    screenState.copy(isProcessing = true)
-                }
-                collectBankAccount(args.clientSecret)
+    fun handlePrimaryButtonClick() {
```

Comment:
This no longer takes `USBankAccountFormScreenState` as an argument (why did it ever?) and only contains logic for the `BillingDetailsCollection` screen state. For any other state, we expect the host surface (PaymentSheet or CustomerSheet) to take over.